GOVERNANCE.md

```diff
@@ -8,11 +8,11 @@ This section describes generic rules and procedures for fulfilling that mandate.
 
 ## Proposing a motion
 
-A maintainer SHOULD propose a motion on the dev@opencontainers.org mailing list (except [security issues](#security-issues)) with another maintainer as a co-sponsor.
+A maintainer SHOULD propose a motion on the <dev@opencontainers.org> mailing list (except [security issues](#security-issues)) with another maintainer as a co-sponsor.
```

Comment:
the `<>` can cause markdown rendering to act like inline HTML. That's why we didn't do this in the past.

Reply:
Markdown lint changed their rules on this which triggered a lot of changes in my code: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md034---bare-url-used

Reply:
@vbatts - is this blocking?

Reply:
nope!